### ISSUE TITLE
[Bug]: The result of DescribeIndex is incomplete when giving one index parameter using default index

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#24532]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.2(2.2.0-20230530-f3fb1950)
- Deployment mode(standalone or cluster): standalone
- MQ type(rocksmq, pulsar or kafka): rocksmq
- SDK version(e.g. pymilvus v2.0.0rc2): 2.2.9.dev25
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

The result of DescribeIndex is incomplete when giving one index parameter using default index.

normal:
```
>>> collection.create_index("float_vector", {})
Status(code=0, message=)
>>> print(collection.index().params)
{'index_type': 'HNSW', 'metric_type': 'IP', 'M': '30', 'efConstruction': '360'}
```

set metric_type L2:
```
>>> collection.create_index("float_vector", {"metric_type": "L2"})
Status(code=0, message=)
>>> print(collection.index().params)
{'metric_type': 'L2'}
```

It's not as expected.

### Expected Behavior

Make the parameters consistent.

### Steps To Reproduce

```markdown
collection.create_index("float_vector", {"metric_type": "L2"})

# check
print(collection.index().params)
print(collection.index().to_dict())
```

**Comment by longjiquan** [comment]
should be fixed by #24534 , please help to recheck this, many thx. /assign @NicoYuan1986

**Comment by NicoYuan1986** [comment]
Fixed.
